manual/custom_codecs/extras/README.md

```diff
@@ -0,0 +1,419 @@
+## Optional codecs
+
+The driver's "extras" module provides additional [codec](../) implementations: these codecs are not required by core
+driver features, but will probably prove useful in a lot of client applications. You can also study their source code as
+a reference to write your own.
+
+The module is published as a separate Maven artifact:
+
+```xml
+<dependency>
+  <groupId>com.datastax.cassandra</groupId>
+  <artifactId>cassandra-driver-extras</artifactId>
+  <version>3.0.0-rc1</version>
```

Comment:
Is there anyway we can parameterize the version here?  Either that or we can put some kind of fake templating here like `DRIVER_VERSION`.